src/styles/card/card.scss

```diff
@@ -0,0 +1,113 @@
+.main {
+  padding: 50px 40px;
+  height: 862px;
+  max-width: 1200px;
+
+  /* Центрування елемента горизонтально */
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  // width: 200px;
+  border: 1px solid white;
```

Comment:
Avoid using colors directly in your SCSS. Instead, define them as variables at the beginning of your file. This makes it easier to manage and maintain your color scheme.